src/audiences/ui/select.js

```diff
@@ -163,8 +163,12 @@ class Select extends Component {
 const applyWithSelect = withSelect( ( select, props ) => {
 	let audiencePost = null;
 
+	// Check if the current user can create audiences. If so, they should have create permissions sent to getPost.
+	const canCreate = select( 'core' ).canUser( 'create', 'audiences' ),
+		queryArgs = canCreate ? { context: 'edit' } : {};
```

Comment:
I think the compiler might take care of optimisations like this now but it's not necessary to assign multiple values from a single `const` declaration. It causes problems with alignment anyway if spaces aren't used. It's usually work ok with `let` or `var` if your tab width was 4 spaces but here we're not gaining much.

```suggestion
	const canCreate = select( 'core' ).canUser( 'create', 'audiences' );
	const queryArgs = canCreate ? { context: 'edit' } : {};
```

Reply:
`¯\_(ツ)_/¯`